receiver/hostmetricsreceiver/internal/scraper/obsreportscraper/obsreportscraper.go

```diff
@@ -0,0 +1,56 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package obsreportscraper
+
+import (
+	"context"
+
+	"go.opencensus.io/trace"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/receiver/hostmetricsreceiver/internal"
+)
+
+type scraper struct {
+	delegate              internal.Scraper
+	scrapeMetricsSpanName string
+}
+
+// newCPUScraper creates a set of CPU related metrics
```

Comment:
Incorrect comment (I actually thought the linter would pick this up since it's an exported method)

Similar for resource scraper

Reply:
Done.